src/toil/common.py

```diff
@@ -132,6 +133,24 @@ def f(x):
         setOption("masterIP") 
         setOption("parasolCommand")
         setOption("maxParasolBatches", int, iC(1))
+
+        def splitEnvVarsToDict(envVars):
+            """
+            Parses of list of environment variables and values. Names and their values are to be
+            separated by a space, an entirely different variable is to be separated by a comma.
+            e.g. --setEnv var1 val1, var2, var3 val3
```

Comment:
1) This is an unconventional syntax. For variables, the convention is `name=value`
2) This syntax does not support spaces in values, which are more common than you might think.
3) argparse has the ability to mention an option multiple times and it will collect all occurrences in a list (see https://docs.python.org/2.7/library/argparse.html#action action='append')